source/agora/consensus/protocol/Nominator.d

```diff
@@ -909,6 +897,16 @@ extern(D):
                 fail_reason, data.prettify);
             return ValidationLevel.kInvalidValue;
         }
+        const Block last_block = this.ledger.getLastBlock();
```

Comment:
We need a check here tho, that slot_idx is the next height, no?

Reply:
Updated